docs/source/tensors.rst

```diff
@@ -177,6 +177,9 @@ Tensor class reference
      use ``tensor.new_*`` creation ops.
 
 .. autoattribute:: Tensor.T
+.. autoattribute:: Tensor.H
+.. autoattribute:: Tensor.mT
+.. autoattribute:: Tensor.mH
```

Comment:
why do we want these attributes?

Reply:
See the OP: https://github.com/pytorch/pytorch/issues/45063#issuecomment-904431478